doc-content/enterprise-only/openshift/readwritemany-proc.adoc

```diff
@@ -0,0 +1,69 @@
+[id='readwritemany-proc']
+= Provisioning persistent volumes with ReadWriteMany access mode using NFS
+
+:rwm-authoring!:
+:rwm-bcmonitoring!:
+
+:rwm-gluster!:
+ifeval::["{context}"=="openshift-operator"]
+:rwm-authoring:
+:rwm-bcmonitoring:
+endif::[]
+ifeval::["{context}"=="openshift-authoring"]
+:rwm-authoring:
+:rwm-gluster:
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+:rwm-authoring:
+:rwm-gluster:
+endif::[]
+
+
+ifdef::rwm-authoring[]
+ifdef::PAM[]
+If you want to deploy 
+ifdef::rwm-bcmonitoring[]
+{CENTRAL} Monitoring, high-availability {CENTRAL},
+endif::rwm-bcmonitoring[]
+ifndef::rwm-bcmonitoring[]
+high-availability {CENTRAL}
+endif::rwm-bcmonitoring[]
+or any {KIE_SERVERS} that use the H2 database, which is the default setting for a non-high-availability authoring environment, your environment must provision persistent volumes with ReadWriteMany access mode.
+endif::PAM[]
+ifdef::DM[]
+If you want to deploy high-availability {CENTRAL}, your environment must provision persistent volumes with ReadWriteMany access mode.
```

Comment:
Search all instances of `ReadWriteMany` and enclose all in tics to make unicode. This is not done consistently, and all should be unicode.